extension/content-script.js

```diff
@@ -2,101 +2,298 @@ console.log("Here is the content script running");
+        </svg>
+        </div>
+        
+        
+        </div>
+        </div>
+        <div class="si-widget-flex-column right">
+        <div class="" style="color:black;font-size:13px;">
+          Widget Status: <input type="checkbox" class="" id="widget-checkbox">
+        </div>
+        </div>
+        </div>
+        <div class="si-widget-container">
+          <div class="event-button" title="Event Info">
+            <div>
+              <svg xmlns="http://www.w3.org/2000/svg" style="width:60px;" viewBox="0 0 448 512"><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M160 80c0-26.5 21.5-48 48-48h32c26.5 0 48 21.5 48 48V432c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V80zM0 272c0-26.5 21.5-48 48-48H80c26.5 0 48 21.5 48 48V432c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V272zM368 96h32c26.5 0 48 21.5 48 48V432c0 26.5-21.5 48-48 48H368c-26.5 0-48-21.5-48-48V144c0-26.5 21.5-48 48-48z"/></svg>
+            </div>
+
+            <span>Event Info <span>
           </div>
 
-          <button class="button">
-              Event Info
-          </button>
-          <button class="button">
-              Cart Info
-          </button>
-          
-      <div>
-      `;
-    console.log("my widge", widget);
-    console.log("updated cart element", cartFormElement.innerHTML);
-    cartFormElement.append(widget);
-  }
+          <div class="cart-button" title="Cart Info">
+            <div>
+              <svg xmlns="http://www.w3.org/2000/svg" style="width:74px;" viewBox="0 0 576 512"><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M0 24C0 10.7 10.7 0 24 0H69.5c22 0 41.5 12.8 50.6 32h411c26.3 0 45.5 25 38.6 50.4l-41 152.3c-8.5 31.4-37 53.3-69.5 53.3H170.7l5.4 28.5c2.2 11.3 12.1 19.5 23.6 19.5H488c13.3 0 24 10.7 24 24s-10.7 24-24 24H199.7c-34.6 0-64.3-24.6-70.7-58.5L77.4 54.5c-.7-3.8-4-6.5-7.9-6.5H24C10.7 48 0 37.3 0 24zM128 464a48 48 0 1 1 96 0 48 48 0 1 1 -96 0zm336-48a48 48 0 1 1 0 96 48 48 0 1 1 0-96z"/></svg>
+            </div>
+            <span>Cart Info<span>
+          </div>
+        </div>
+        </div>
+        
+    <div>
+    `;
+  console.log("my widge", widget);
+  console.log("updated cart element", cartFormElement.innerHTML);
+  cartFormElement.append(widget);
 }
 
 function configureWidgetCheckbox() {
   const widgetCheckbox = document.querySelector("#widget-checkbox");
-  if (widgetCheckbox) {
-    const widgetDisabled = localStorage.getItem("disable-shopify-widget");
-    widgetCheckbox.checked = !!!widgetDisabled;
-    widgetCheckbox.onchange = function (e) {
-      localStorage.setItem("disable-shopify-widget", !e.target.checked);
-    };
-  }
+  const widgetDisabled = localStorage.getItem("disable-shopify-widget");
+  widgetCheckbox.checked = !!!widgetDisabled;
```

Comment:
No need to use !!!, just single ! is enough

Reply:
Resolved